docs/notebook/Tutorial.ipynb

```diff
@@ -528,8 +528,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Interpolations are evaluated lazily on field access.<br/>\n",
-    "Note below that when printed the interpolations are not resolved.<br/>\n",
+    "Note below that when printed the interpolations are resolved.<br/>\n",
     "They get resolved once you access them."
```

Comment:
Interpolations are evaluated lazily on field access. OmegaConf.to_yaml() is eagerly resolving the interpolations by default. You can change it by passing resolve=False.

Example without resolve=False.
Example with resolve=False

Reply:
The other related doc update here is also similarly confusing.